File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.swift
import Foundation

func solution(_ numbers:[Int]) -> [Int] {
    var sumArr: [Int] = []
    
    for i in 0..<numbers.count {
        for j in i+1..<numbers.count {
            sumArr.append(numbers[i] + numbers[j])
        }
    }
    return Array(Set(sumArr)).sorted()
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.*;
class Solution {
    public int[] solution(int[] numbers) {
        Set<Integer> sums = new HashSet<>();

        for (int i = 0; i < numbers.length - 1; i++) {
            for (int j = i + 1; j < numbers.length; j++) {
                sums.add(numbers[i] + numbers[j]);
            }
        }
        
        int[] answer = new int[sums.size()];
        int idx = 0;
        for (int num: sums) {
            answer[idx++] = num;
        }
        
        Arrays.sort(answer);
        return answer;
         
    }
}